samples/BuildMinimalMaterialAppCompat/ClassLibrary/ClassLibrary.csproj

```diff
@@ -19,6 +19,8 @@
     <AndroidResgenFile>Resources\Resource.designer.cs</AndroidResgenFile>
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
     <TargetFrameworkVersion>v12.0</TargetFrameworkVersion>
+    <AndroidApplication>True</AndroidApplication>
```

Comment:
What is this for? It does not seem correct. A class library should not be an `AndroidApplication`.